packages/tasks/src/tasks/any-to-any/data.ts

```diff
@@ -1,10 +1,12 @@
 import type { TaskDataCustom } from "..";
 
 const taskData: TaskDataCustom = {
-	datasets: [{
-			description:  "All-Modality Alignment Dataset with Fine-grained Preference Annotations and Language Feedback.",
+	datasets: [
+		{
+			description: "All-Modality Alignment Dataset with Fine-grained Preference Annotations and Language Feedback.",
 			id: "PKU-Alignment/align-anything-400k",
-		},],
+		},
+	],
 	demo: {
 		inputs: [],
```

Comment:
would you like to add a demo as well? maybe image/text input to image/text output